docs/products/kafka/kafka-connect/howto/elasticsearch-sink.rst

```diff
@@ -0,0 +1,243 @@
+    * ``value.converter.basic.auth.credentials.source``: to the value ``USER_INFO``, since you're going to login to the schema registry using username and password.
+    * ``value.converter.schema.registry.basic.auth.user.info``: passing the required schema registry credentials in the form of ``SCHEMA_REGISTRY_USER:SCHEMA_REGISTRY_PASSWORD`` with the ``SCHEMA_REGISTRY_USER`` and ``SCHEMA_REGISTRY_PASSWORD`` parameters :ref:`retrieved in the previous step <connect_elasticsearch_sink_prereq>`. 
+
+
+Create a Kafka Connect connector with Aiven Console
+'''''''''''''''''''''''''''''''''''''''''''''''''''
+
+To create the connector, access the `Aiven Console <https://console.aiven.io/>`_ and select the Aiven for Apache Kafka® or Aiven for Apache Kafka Connect® service where the connector needs to be defined, then:
+
+* Click on the **Connectors** tab
+* Clink on **Create New Connector**, the button is enabled only for services :doc:`with Kafka Connect enabled <enable-connect>`.
+* Select the **Elasticsearch sink**
+* Under the *Common* tab, locate the **Connector configuration** text box and click on **Edit**
+* Paste the connector configuration (stored in the ``elasticsearch_sink.json`` file) in the form
+* Click on **Apply**
+
+.. Note::
+
+    The Aiven Console parses the configuration file and fills the relevant UI fields. You can review the UI fields across the various tab and change them if necessary. The changes will be reflected in JSON format in the **Connector configuration** text box.
+
+* After all the settings are correctly configured, click on **Create new connector**
+* Verify the connector status under the **Connectors** tab
+* Verify the presence of the data in the target Elasticsearch service, the index name is equal to the Apache Kafka topic name
+
+.. Note::
+
+    Connectors can be created also using the dedicated :ref:`Aiven CLI command <avn_service_connector_create>`.
+
+Create daily Elasticsearch indexes
+----------------------------------
+
+You might need to create a new Elasticsearch index on daily basis to store the Apache Kafka messages. 
+Adding the following ``TimestampRouter`` transformation in the connector properties file provides a way to define the index name as concatenation of the topic name and message date.
+
+::
+
+    "transforms": "TimestampRouter",
+    "transforms.TimestampRouter.topic.format": "${topic}-${timestamp}",
+    "transforms.TimestampRouter.timestamp.format": "yyyy-MM-dd",
+    "transforms.TimestampRouter.type": "org.apache.kafka.connect.transforms.TimestampRouter"
+
+.. Warning::
+
+    The current version of the Elasticsearch sink connector is not able to automatically create daily indexes in Elasticsearch. Therefore you need to create the indexes with the correct name before starting the sink connector. You can create Elasticsearch indexes in many ways including :doc:`CURL commands </docs/products/opensearch/howto/opensearch-with-curl>`.
```

Comment:
There are 8 places in the PR with indexes - Let's use "indices" everywhere. Since the docs use them https://opensearch.org/docs/latest/opensearch/rest-api/cat/cat-indices/ https://www.elastic.co/guide/en/elasticsearch/reference/current/cat-indices.html

Reply:
fixed